src/visitor.rs

```diff
@@ -601,6 +619,7 @@ impl<'b, 'a: 'b> FmtVisitor<'a> {
         config: &'a Config,
         snippet_provider: &'a SnippetProvider<'_>,
         report: FormatReport,
+        base_skip_macro_names: Vec<String>,
```

Comment:
Can you accept a slice instead, e.g, `&[String]` ? That way, I think you can avoid the call to `#clone` in https://github.com/rust-lang/rustfmt/pull/3454/files#diff-85974d8eb6610bd6ea773d375ad8c6d2R611

Reply:
base_skip_macro_names is gone.
related: https://github.com/rust-lang/rustfmt/pull/3454#discussion_r267381139

Reply:
ah ok sorry about that